build.gradle

```
@@ -19,14 +26,44 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-mail'
 	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
-	implementation 'org.flywaydb:flyway-core'
+	implementation 'org.flywaydb:flyway-core:10.11.1'
+	implementation 'org.flywaydb:flyway-database-postgresql:10.11.1'
+
+	implementation('com.fasterxml.jackson.module:jackson-module-afterburner')
```

```suggestion
	implementation 'com.fasterxml.jackson.module:jackson-module-afterburner'
```
Seria legal manter o padrão das outras libs.